AAI/AAIConnectProfile.md

```diff
@@ -511,128 +503,169 @@ JWTs follow <https://tools.ietf.org/html/rfc7515> (JWS).
+Only the GA4GH claims truly must be as prescribed here. Refer to OIDC Spec for
+more information. The /userinfo endpoint MAY use `application/json` or
+`application/jwt`. If `application/jwt` is returned, it MUST be signed as per
+[UserInfo](https://openid.net/specs/openid-connect-core-1_0.html#UserInfoResponse).
+
 ```
 {
- "iss": "https://\<issuer website\>/",
- "sub": "<someone@someone.com>",
- "idp": "google",
+ "iss": "https://<issuer-website>/",
+ "sub": "<subject-identifier>",
  "aud": [
-  "client_id",
-  "client_id2"
+  "<client-id1>",
+  "<client-id2>" ...
  ],
- "iat": 1553545136,
- "exp": 1553631536,
  <ga4gh-spec-claims>
 }
 ```
--   `<ga4gh-spec-claims>`: Claims included as part of a GA4GH standard specification based on the scopes provided. This content MAY be incomplete (i.e. a subset of data elements) and more may be fetched as indicated by ga4gh_userinfo_claims. A non-normative example of `<ga4gh-spec-claims>` is: "ga4gh": {[ga4gh claims](https://github.com/ga4gh-duri/ga4gh-duri.github.io/blob/master/researcher_ids/RI_Claims_V1.md)}
 
-#### Embedded Token issued by Embedded Claim Signatory
+-   `iss` and `sub`: REQUIRED.
+
+-   `aud`: OPTIONAL.
 
-There are two supported formats for Embedded Tokens:
+-   `<ga4gh-spec-claims>`: OPTIONAL. GA4GH Claims are generally included as
+    part of one or more GA4GH standard specifications based on the scopes
+    provided. Even when requested by the appropriate scopes, these GA4GH Claims
+    may not be included in the response for various reasons, such as if the
+    user does not have any GA4GH Claims. See
+    [Authorization/Claims](#authorizationclaims) for an example of a GA4GH
+    Claim.
 
-1.   Conforms with the [Access Token](#access-token-issued-by-broker) format,
-     and MUST NOT contain a `jku` in the header. However, MUST NOT include
-     `aud`, `idp`, `scope`, `ga4gh_userinfo_claims`.
+#### Embedded Token issued by Embedded Token Signatory
+
+There are two supported formats for Embedded Tokens.
+
+##### Embedded Access Token Format
+
+Follows the [Broker access token](#access-token-issued-by-broker) format
+with the following caveats:
      
-2.   Follows the [Access Token](#access-token-issued-by-broker) format, except
-     it does contain a `jku` in the header and MUST NOT have "openid" in the
-     `scope` claim. When a `jku` is present, only the following headers and
-     claims are REQUIRED:
-
-     ```
-     {
-       "typ": "JWT",
-       "alg": "RS256",
-       "jku": "https://\<jwk URL\>",
-       "kid": "xxxxx"
-     }.
-     {
-       "iss": "https://\<issuer website\>/",
-       "sub": "\<subject identifier\>",
-       "iat": 1553545136,
-       "exp": 1553631536
-     }.
-     \<signature\>
-     ```
+1.  MUST NOT contain a `jku` in the header.
+
+2.  The payload claims MUST NOT include `aud`.
```

Comment:
The part "Access_token issued by Broker" part makes "aud" claim REQUIRED, while this part makes it forbidden. Both is not possible.

Reply:
Also, having the "aud" (audience) claim in access tokens forces the issuing broker to all possible audiences in the moment of issuing the token, which is very unlikely if the access token is expected to be passed down to a chain of brokers.

Reply:
I think this might be poor language. The "access token issued by Broker" says 
```
aud: MUST contain the Oauth Client ID of the relying party except when allowing the token to be used as an open federation Embedded Token where aud MUST NOT be included. When included, it MAY contain other strings or identifiers as well.
```
I think this should be made to be clear that it's OPTIONAL (May) as per https://tools.ietf.org/html/rfc7519#section-4 BUT if included must conform to our spec. 

As far as the embedded tokens goes I'd say leave it as OPTIONAL too. 

As long as the client is checking SOMETHING we're in good shape. Often/usually that's `aud` but doesn't have to be. `iss` is fine, too. 

My vote is to make the `Access_token issued by Broker` clear that this can be optional as long as we then make sure the client specification really hammers down that the clients have to check something.

Reply:
At https://github.com/ga4gh/data-security/blob/f7694d7df96f807438075c3de552fc70b10036ab/AAI/AAIConnectProfile.md#access_token-issued-by-broker I see:
```
aud: REQUIRED. MUST contain the Oauth Client ID of the relying party.
```
Maybe you see some other version? Your wording seems reasonable.

Reply:
Yes -- I'm looking at the one in Master. @cdvoisin can we keep it as Optional without breaking anything?

Reply:
Fixed